Resources/Documentation/installation_and_setup.md

```diff
@@ -0,0 +1,88 @@
+## Installing vrecord
+
+If you use macOS, you can easily install vrecord using a package manager called Homebrew. To install Homebrew, follow [these instructions](https://brew.sh/).
```

Comment:
>If you use macOS

It‘s not needed, as the classic `vrecord` is running only under macOS.

Reply:
i suggest this is not blocking, content changes could be in a subsequent pr